### PR TITLE
update mlp of llama

### PR DESCRIPTION
**Pull request by rnwang04**
## Description

use `xe_addons.mlp_silu_mul_inplaced(gate, up)` for llama series model

### 4. How to test?
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.

### CODE REVIEW

**Review by jason-dai — APPROVED** [review]
LGTM

**Review by MeouSker77 — APPROVED** [review]

**Comment by rnwang04** [comment]
Well, PR validation is failed: https://github.com/intel-analytics/ipex-llm-workflow/actions/runs/10505369582/job/29104277776. Need a further look.

**Comment by rnwang04** [comment]
`xe_addons.mlp_silu_mul_inplaced(gate, up)` may cause output logits changes a little.
e.g. the max_diff_tensor for Mistral-7B-Instruct is [0.0009765625].
Here I expand the lower_bound threshold to 1e-3 first to make our UT can pass.

**Comment by rnwang04** [comment]
PR validation is passed: https://github.com/intel-analytics/ipex-llm-workflow/actions/runs/10506853978
